pom.xml

```diff
@@ -1,70 +1,229 @@
-		</developer>
-	</developers>
-
-	<scm>
-		<connection>scm:git:git://github.com/NetCal/DiscoDNC.git</connection>
-		<developerConnection>scm:git:ssh://github.com/NetCal/DiscoDNC.git</developerConnection>
-		<url>https://github.com/NetCal/DiscoDNC/tree/master</url>
-	</scm>
-	
-	<properties>
-        	<maven.compiler.source>1.8</maven.compiler.source>
-        	<maven.compiler.target>1.8</maven.compiler.target>
-        	<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-	</properties>
-
-	<dependencies>
-
-		<dependency>
-			<groupId>org.apache.commons</groupId>
-			<artifactId>commons-math3</artifactId>
-			<version>[3.0,3.6.1]</version>
-		</dependency>
-
-	</dependencies>
-	
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>discodnc.cs.uni-kl.de</groupId>
+    <version>2.5.0</version>
+    <name>DiscoDNC - The Disco Deterministic Network Calculator</name>
+    <description>Deterministic Network Calculus (DNC) is a methodology for worst-case modeling and analysis of
+        communication networks. It enables to derive deterministic bounds on a server’s backlog as well as a flow’s
+        end-to-end delay. Given a directed graph of servers (server graph) and the flows crossing these servers, the
+        Disco Deterministic Network Calculator (DiscoDNC) automates the derivation of bounds.
+    </description>
+    <url>discodnc.cs.uni-kl.de</url>
+
+    <artifactId>DiscoDNC</artifactId>
+
+    <licenses>
+        <license>
+            <name>GNU General Lesser Public License (LGPL) version 2.1</name>
+            <url>https://www.gnu.org/licenses/lgpl-2.1.txt</url>
+        </license>
+    </licenses>
+
+    <developers>
+        <developer>
+            <name>Steffen Bondorf</name>
+            <email>bondorf@cs.uni-kl.de</email>
+            <organization>Distributed Computer Systems (DISCO) Lab at TU Kaiserslautern</organization>
+            <organizationUrl>https://disco.cs.uni-kl.de</organizationUrl>
+        </developer>
+        <developer>
+            <name>Philipp Schon</name>
+            <email>schon@cs.uni-kl.de</email>
+            <organization>Distributed Computer Systems (DISCO) Lab at TU Kaiserslautern</organization>
+            <organizationUrl>https://disco.cs.uni-kl.de</organizationUrl>
+        </developer>
+        <developer>
+            <name>Zoltan Matyas</name>
+            <email>matyesz79@gmail.com</email>
+            <organization></organization>
+            <organizationUrl></organizationUrl>
+        </developer>
+    </developers>
+
+    <scm>
+        <connection>scm:git:git://github.com/NetCal/DiscoDNC.git</connection>
+        <developerConnection>scm:git:ssh://github.com/NetCal/DiscoDNC.git</developerConnection>
+        <url>https://github.com/NetCal/DiscoDNC/tree/master</url>
+    </scm>
```

Comment:
This is tricky because we include all other projects as submodules. In this case I would list all the repos here.